WalletWasabi.Fluent/ViewModels/Dialog/DialogViewModelBase<TResult>.cs

```diff
@@ -0,0 +1,52 @@
+using System;
```

Comment:
The name of this file should be changed, this what I got when trying to checkout the branch:

```
$ git checkout fluent/popup-infra
error: unable to create file WalletWasabi.Fluent/ViewModels/Dialog/DialogViewModelBase<TResult>.cs: Invalid argument
Switched to branch 'fluent/popup-infra'
D       WalletWasabi.Fluent/ViewModels/Dialog/DialogViewModelBase<TResult>.cs
```

Reply:
ah that's why it's failing on windows, what can you suggest for a naming convention for our generic type classes?

Reply:
i settled with `OfT` postfix naming convention. let's change it when we have any consensus about it.